learn2track/models/ffnn_regression.py

```diff
@@ -55,7 +55,7 @@ def __init__(self, volume_manager, input_size, hidden_sizes, output_size, activa
 
         if self.dropout_prob:
             p = 1 - self.dropout_prob
-            self.dropout_vectors[self.layer_regression.name] = self.srng.binomial(size=(self.layer_regression.W.shape[0],), n=1, p=p, dtype=floatX) / p
+            self.dropout_vectors[self.layer_regression.name] = self.srng.binomial(size=(self.layer_regression.input_size,), n=1, p=p, dtype=floatX) / p
```

Comment:
If I understand correctly, for `DenseLayer` we are still dropping the input whereas `GruLayer` and `LSTMLayer` we are dropping the hidden states?